### PR TITLE
Adding more explicit contirbutors and institutional partners

### PR DESCRIPTION
**Pull request by choldgraf**
This makes 2i2c's institutional partners more explicit. The goals are to provide credit to the institutions that are directly or indirectly helping 2i2c happen, and make sure that they are seen as providing partnership and value to 2i2c. It does the following:

- Adds a ["Funders and Collaborators" section](https://deploy-preview-33--cocky-booth-e7ed17.netlify.app/about/#collaborators-and-funders), This lists our direct funding, as well as the "Collaborating Institutions" of 2i2c.
- Adds the institutional affiliation of founders to [the founding team page](https://deploy-preview-33--cocky-booth-e7ed17.netlify.app/founders/#founders), to draw the connections more strongly.

cc the founders since I want to make sure this looks good to you:

@rabernat am I describing Lamont etc correctly? (and do you think it makes sense to use Lamont and not "Pangeo" here?)
@yuvipanda @fperez @clcarson Are you OK with your affiliations? Do you think instead I should use department-specific affiliations? Or is it OK to just use CDSS as a blanket org?
@lheagy let me know when I should change this over to UBC (:tada:)
@colliand I assume that PIMS is correct as a primary institutional partner, not UBC

More generally, please provide any feedback (most helpful would be suggested edits to this PR) to improve this.

Once I get 👍 from folks then I will merge in.

### CODE REVIEW

**Comment by clcarson** [comment]
I think this is somewhat delicate.

On the collaborators and funders page, I would leave out the filler sentence "A range of funders, academic institutions, and individuals contribute to 2i2c in different ways.", "We are grateful to have formal funding support from several funders.," and "These institutions devote significant resources to 2i2c (in alphabetical order). We are always looking to expand this list." To my eye they're not necessary, and the last is troublesome saying that they devote resources to 2i2c. We haven't got that nailed down yet.

Less is more, from my point of view, about the funders, particularly since they're heterogeneous. Is it essential to say the amounts and give details? Would it work to just link to the blog posts.

For CDSS, I would simply say "CDSS pioneered efforts in deploying JupyterHub infrastructure for education that inspired 2i2c’s creation and is a partner in its educational outreach efforts." The rest will get us in trouble. I would ask Fernando to check the text with Kathy before putting it up.

"The" should be capitalized for ICSI.

In each of the other institutional entries, I would leave out "It provides strategic guidence and support, and provided founding support for 2i2c." That may put our institutions on the hook for something they haven't formally committed.

For founders, I missed the creation of the advisory board. :-) Is that the right term?

Personally, I'd put the pictures of us at the bottom, after "Our experience." I think "Our experience" does a better job in making clear what we bring."I would just say "2i2c team members are core contributors or project leads on the following services and tools."

University of California, Berkeley is the official name of UC Berkeley. This is the right level, not CDSS. (I'm not in CDSS.)

**Comment by choldgraf** [comment]
re: the language, I was mostly just emulating what Pangeo had done [on their collaborators page](https://pangeo.io/collaborators.html) as an example of what a successful project has done for this kind of page.

hmm, if people think there is downside to being more verbose, I can just remove all of the language about each entirely and see how that looks. I'd like to have some kind of "minimal viable page" up so don't want this to take a long time if there's controversial language in there.

**Comment by choldgraf** [comment]
> For founders, I missed the creation of the advisory board. :-) Is that the right term?

I have no opinion here, I just wanted a word to describe people's relationship with the organization right now other than "founder", and that could be included along with people that are paid staff w/ titles. I'm happy with whatever if there's a better term.

**Comment by choldgraf** [comment]
I've just pushed a change that:

- Removes the extra wording
- Moves the collaborators page instead to a *subsection* of the "about us" page
- Calls it "collaborating institutions"
- Fixes the institutional name changes @clcarson suggested

File: content/about/collaborators.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <a class="collaborator-link" href="https://www.ldeo.columbia.edu/">
+    <img src="https://pangeo.io/_images/ldeo-logo.png" alt="Lamont Doherty Earth Observatory" />
+  </a>
+  </div>
+
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://www.pims.math.ca/">
+    <img src="https://ssc.ca/sites/default/files/imce/webhorizfulllarge.png" alt="Pacific Institute for Mathematical Sciences" />
+  </a>
+  </div>
```
Comment (lheagy) [inline content/about/collaborators.md:63]: It is probably appropriate to add UBC here now for both @colliand and I -- thoughts @colliand??

Comment (colliand) [reply content/about/collaborators.md:63]: Yes. I agree. Let's list UBC and PIMS.

**Comment by choldgraf** [comment]
I'm going to close this one as it's out of date now, and it's unclear what implications come with listing institutions on our home page. I still think that we should do this, but should probably think it through a little bit before posting it live. We can tackle that in a follow-up.

**Pull request closed without merging**